src/errors/AuthFetchError.js

```diff
@@ -0,0 +1,9 @@
+class AuthFetchError extends Error {
+    constructor(message, response, body) {
+        super(message)
+        this.response = response
+        this.body = body
+    }
+}
+
+module.exports = AuthFetchError
```

Comment:
module.exports?

Reply:
Wanted to make it similar to other errors. Changed to `export default` now. It'd be good to be consistent here tho. It's a good material for a future refactoring.